index.js

```diff
@@ -1,13 +1,38 @@
+// Use UTC for for all Date parsing
+process.env.TZ = 'UTC';
+
 const fs = require('fs');
+const parseReminder = require('parse-reminder');
 const createScheduler = require('probot-scheduler');
+const commands = require('probot-commands');
 const Freeze = require('./lib/freeze');
 const formatParser = require('./lib/format-parser');
 const githubHelper = require('./lib/github-helper');
 
 /* Configuration Variables */
 
 module.exports = robot => {
+  commands(robot, 'remind', async (context, command) => {
+    const reminder = parseReminder(command.name + ' ' + command.arguments);
+
+    if (reminder) {
+      if (reminder.who === 'me') {
+        reminder.who = context.payload.comment.user.login;
+      }
+
+      const config = await context.config('probot-snooze.yml', JSON.parse(fs.readFileSync('./etc/defaults.json', 'utf8')));
+      const freeze = new Freeze(context.github, config);
+
+      freeze.freeze(context, {
+        assignee: reminder.who,
+        unfreezeMoment: reminder.when,
+        message: reminder.what
+      });
```

Comment:
reminder.[who|what|when] is AWESOME

Reply:
👍  I was planning to open another PR that renames the values that are stored to that.